manager/integration/tests/test_scheduling.py

```diff
@@ -695,6 +689,8 @@ def test_data_locality_basic(client, core_api, volume_name, pod, settings_reset)
     volume2_rep1 = volume2_healthy_replicas[0]
     volume2_rep2 = volume2_healthy_replicas[1]
     assert volume2_rep1["hostId"] != volume2_rep2["hostId"]
+    delete_and_wait_pod(core_api, pod2_name)
+    wait_for_volume_detached(client, volume2_name)
```

Comment:
Make sure to clean up the pod after we done with it. Otherwise, PVC cannot be removed at the in the finalized at the end of the test.